### PR TITLE
feat(nuxt): Add downgrade path to nitro 2.9.7

### PR DESCRIPTION
**Pull request by andreiborza**
Screenshot 2024-11-29 at 10 39 22@2x

See: https://github.com/getsentry/sentry-javascript/issues/14514

### CODE REVIEW

**Review by AbhiPrasad — APPROVED** [review]

File: src/utils/clack-utils.ts [hunk 1]
```diff
@@ -1469,3 +1491,18 @@ export async function featureSelectionPrompt<F extends ReadonlyArray<Feature>>(
     return selectedFeatures as { [key in F[number]['id']]: boolean };
   });
 }
+
+export async function askShouldAddNuxtOverride(
```
Comment (Lms24) [inline src/utils/clack-utils.ts:1495]: l: given this was added to clack-utils, let's try to get rid of the nuxt-specific part.

```suggestion
export async function askShouldAddPackageOverride(
```

File: src/utils/clack-utils.ts [hunk 2]
```diff
@@ -1469,3 +1491,18 @@ export async function featureSelectionPrompt<F extends ReadonlyArray<Feature>>(
...
+  pkgName: string,
+  pkgVersion: string,
+): Promise<boolean> {
+  return traceStep(`ask-add-nuxt-overrides`, () =>
```
Comment (Lms24) [inline src/utils/clack-utils.ts:1499]:
```suggestion
  return traceStep(`ask-add-override`, () =>
```

**Review by Lms24 — APPROVED** [review]

File: src/utils/clack-utils.ts (hunk 1 above)
Comment (andreiborza) [reply src/utils/clack-utils.ts]: Right, updated!

File: src/utils/clack-utils.ts (hunk 2 above)
Comment (andreiborza) [reply src/utils/clack-utils.ts]: Named it `ask-add-package-override`

**Pull request merged**